docs/.vitepress/vitepress/components/vp-subnav.vue

```diff
@@ -12,13 +12,13 @@ const { shouldShow, scrollToTop } = useBackTop()
   <div class="sub-nav py-3 flex items-center">
     <ToggleSidebarBtn v-if="hasSidebar" @click="$emit('open-menu')" />
     <Transition name="shifting">
-      <ElButton
-        type="text"
+      <ElLink
+        :class="{ 'go-back-top': true, show: shouldShow }"
+        :underline="false"
         class="height-5"
         size="small"
-        :class="{ 'go-back-top': true, show: shouldShow }"
         @click.prevent.stop="scrollToTop"
-        >{{ 'Back to top' }}</ElButton
+        >{{ 'Back to top' }}</ElLink
       >
```

Comment:
```suggestion
        >
          {{ 'Back to top' }}
        </ElLink>
```

Reply:
The linter did not say anything about this, I guess we should do something about the linter/formatter?

Reply:
Hmm we can consider it later.